h/feeds/util.py

```diff
@@ -1,11 +1,27 @@
 """Utility functions for feed-generating code."""
+from datetime import timedelta
+from datetime import tzinfo
+
 from h._compat import urlparse
 
 # See RFC4151 for details of the use and format of the tag date:
 #
 #   https://tools.ietf.org/html/rfc4151#section-2.1
 FEED_TAG_DATE='2015-09'
 
+ZERO = timedelta(0)
+
+
+class UTC(tzinfo):
```

Comment:
I'd suggest that instead of converting a naive datetime into a timezone-aware datetime, just change the format.